sympy/sets/conditionset.py

```diff
@@ -246,6 +246,37 @@ def _eval_subs(self, old, new):
             return ConditionSet(new, Contains(new, base), base)
         return self.func(self.sym, cond, base)
 
+    def _as_interval(self):
+        """`as_interval` is only implemented for univariate conditions and real base sets"""
+        if not isinstance(self.sym, Tuple) and  self.base_set.is_subset(S.Reals):
+            try:
+                return self.condition.as_set().intersect(self.base_set)
+            except NotImplementedError:
+                return self
+        return self
+
+    @staticmethod
+    def _parse_condition_sets(*sets):
+        return tuple(s._as_interval() if isinstance(s, ConditionSet) else s for s in sets)
+
+    def _eval_is_subset(self, other):
+        s, o = self._parse_condition_sets(self, other)
+
+        s_o = s.intersect(o)
+        if s_o == s:
+            return True
+        elif isinstance(s_o, Intersection):
+            return None
+        else:
+            return False
```

Comment:
Isn't this code in the superclass is_subset method? I don't think it needs to be repeated here. In any case it is broken logic. We can't conclude anything from whether `s_o` is an intersection.

Reply:
Yes, the code is identical. I kept getting recursion errors when trying to call the superclass method in a semi-decent way (alternatively could just return `None` in this method, which would continue execution of the superclass method) which is why I copied the logic. Would appreciate suggestions on how to implement this nicely

Reply:
Nvm, I just force-pushed, see if it's better now